### ISSUE TITLE
IllegalStateException - days of NULL

### ISSUE THREAD

**Issue by x-yingcan-x** [SkyblockerMod/Skyblocker#919]
### Minecraft Version

1.21.0/1.21.1

### Skyblocker Version

skyblocker-1.21.1-beta-ac08fb6+1.21.1

### Reproduction Steps

1. hover on blaze slayer lv1 reward (screenshot from hyp wiki)
![图片](https://github.com/user-attachments/assets/7a4af721-45f6-4561-bafc-950663539afe)
2. crash

### Crash Report file and latest.log

```shell
---- Minecraft Crash Report ----
// Daisy, daisy...

Time: 2024-08-13 21:14:01
Description: Rendering screen

java.lang.IllegalStateException: Not a JSON object: null
	at com.mojang.serialization.DataResult$Error.getOrThrow(DataResult.java:287)
	at com.mojang.serialization.DataResult.getOrThrow(DataResult.java:81)
	at de.hysky.skyblocker.utils.ItemUtils.getNeuId(ItemUtils.java:257)
	at net.minecraft.class_1799.getNeuName(class_1799.java:5235)
	at de.hysky.skyblocker.skyblock.item.tooltip.adders.AvgBinTooltip.addToTooltip(AvgBinTooltip.java:22)
	at de.hysky.skyblocker.skyblock.item.tooltip.TooltipManager.addToTooltip(TooltipManager.java:90)
	at de.hysky.skyblocker.skyblock.item.tooltip.TooltipManager.lambda$init$0(TooltipManager.java:53)
	at net.fabricmc.fabric.api.client.item.v1.ItemTooltipCallback.lambda$static$0(ItemTooltipCallback.java:35)
	at net.minecraft.class_1799.handler$zkn000$fabric-item-api-v1$getTooltip(class_1799.java:3636)
	at net.minecraft.class_1799.method_7950(class_1799.java:807)
	at net.minecraft.class_437.method_25408(class_437.java:243)
	at net.minecraft.class_465.method_51454(class_465.java:187)
	at net.minecraft.class_465.method_2380(class_465.java:182)
	at net.minecraft.class_476.method_25394(class_476.java:28)
	at net.minecraft.class_437.method_47413(class_437.java:117)
	at net.minecraft.class_757.mixinextras$bridge$method_47413$224(class_757.java)
	at net.minecraft.class_757.wrapOperation$bck000$fabric-screen-api-v1$onRenderScreen(class_757.java:4086)
	at net.minecraft.class_757.method_3192(class_757.java:913)
	at net.minecraft.class_310.method_1523(class_310.java:1285)
	at net.minecraft.class_310.method_1514(class_310.java:882)
	at net.minecraft.client.main.Main.main(Main.java:256)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:71)
	at oolloo.jlw.Wrapper.main(Wrapper.java:51)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at com.mojang.serialization.DataResult$Error.getOrThrow(DataResult.java:287)
	at com.mojang.serialization.DataResult.getOrThrow(DataResult.java:81)
	at de.hysky.skyblocker.utils.ItemUtils.getNeuId(ItemUtils.java:257)
	at net.minecraft.class_1799.getNeuName(class_1799.java:5235)
	at de.hysky.skyblocker.skyblock.item.tooltip.adders.AvgBinTooltip.addToTooltip(AvgBinTooltip.java:22)
	at de.hysky.skyblocker.skyblock.item.tooltip.TooltipManager.addToTooltip(TooltipManager.java:90)
	at de.hysky.skyblocker.skyblock.item.tooltip.TooltipManager.lambda$init$0(TooltipManager.java:53)
	at net.fabricmc.fabric.api.client.item.v1.ItemTooltipCallback.lambda$static$0(ItemTooltipCallback.java:35)
	at net.minecraft.class_1799.handler$zkn000$fabric-item-api-v1$getTooltip(class_1799.java:3636)
	at net.minecraft.class_1799.method_7950(class_1799.java:807)
	at net.minecraft.class_437.method_25408(class_437.java:243)
	at net.minecraft.class_465.method_51454(class_465.java:187)
	at net.minecraft.class_465.method_2380(class_465.java:182)
	at net.minecraft.class_476.method_25394(class_476.java:28)
	at net.minecraft.class_437.method_47413(class_437.java:117)
	at net.minecraft.class_757.mixinextras$bridge$method_47413$224(class_757.java)
	at net.minecraft.class_757.wrapOperation$bck000$fabric-screen-api-v1$onRenderScreen(class_757.java:4086)

-- Screen render details --
Details:
	Screen name: net.minecraft.class_476
	Mouse location: Scaled: (293, 145). Absolute: (587.000000, 291.000000)
	Screen size: Scaled: (683, 384). Absolute: (1366, 768). Scale factor of 2.000000
Stacktrace:
	at net.minecraft.class_757.method_3192(class_757.java:913)
	at net.minecraft.class_310.method_1523(class_310.java:1285)
	at net.minecraft.class_310.method_1514(class_310.java:882)
	at net.minecraft.client.main.Main.main(Main.java:256)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:71)
	at oolloo.jlw.Wrapper.main(Wrapper.java:51)

-- Uptime --
Details:
	JVM uptime: 22025.014s
	Wall uptime: 21975.185s
	High-res time: 116022.918s
	Client ticks: 438778 ticks / 21938.900s
Stacktrace:
	at net.minecraft.class_310.method_1587(class_310.java:2501)
	at net.minecraft.class_310.method_54580(class_310.java:949)
	at net.minecraft.class_310.method_1514(class_310.java:902)
	at net.minecraft.client.main.Main.main(Main.java:256)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:71)
	at oolloo.jlw.Wrapper.main(Wrapper.java:51)

-- Affected level --
Details:
	All players: 8 total; [class_746['Pqwq'/7782004, l='ClientLevel', x=35.30, y=100.00, z=3.30], class_745['sohpiaa'/7782007, l='ClientLevel', x=35.28, y=112.07, z=3.28], class_745['§bSam '/7782181, l='ClientLevel', x=8.50, y=100.00, z=41.50], class_745['Scion '/7791234, l='ClientLevel', x=9.56, y=100.00, z=-11.94], class_745['Scion '/7990148, l='ClientLevel', x=13.13, y=100.00, z=-9.00], class_745['Scion '/8092403, l='ClientLevel', x=10.03, y=100.00, z=-10.31], class_745['Scion '/8184776, l='ClientLevel', x=9.50, y=100.00, z=-9.06], class_745['Scion '/8278041, l='ClientLevel', x=13.53, y=100.00, z=-9.91]]
	Chunk stats: 16384, 225
	Level dimension: minecraft:overworld
	Level spawn location: World: (7,100,7), Section: (at 7,4,7 in 0,6,0; chunk contains blocks 0,0,0 to 15,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 31126 game time, 40754 day time
	Server brand: Hypixel BungeeCord (2024.8.7.1) <- Hygot 2024.8.9.2
	Server type: Non-integrated multiplayer server
	Tracked entity count: 181

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: vanilla, fabric, aaron-mod, async-pack-scan, betterconfig, caffeineconfig, clientcommands, cloth-config, conditional-mixin, dynamic_fps, fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-blockrenderlayer-v1, fabric-client-tags-api-v1, fabric-command-api-v1, fabric-command-api-v2, fabric-commands-v0, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-convention-tags-v2, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-game-rule-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-keybindings-v0, fabric-language-kotlin, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-loot-api-v3, fabric-message-api-v1, fabric-model-loading-api-v1, fabric-networking-api-v1, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-renderer-registries-v1, fabric-rendering-data-attachment-v1, fabric-rendering-fluids-v1, fabric-rendering-v0, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-sound-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader, faster-random, fastitems, figura, freelook, fzzy_config, higher-chat, hm-api, jjpizza, libjf, libjf-config-core-v2, libjf-config-network-v0, libjf-config-ui-tiny, libjf-data-v0, libjf-translate-v1, libjf-web-v1, libmcdev-fabric, libzoomer, lithium, malilib, methane, midnightlib, modernfix, modmenu, moreculling, motionblur, ok_zoomer, org_apache_commons_commons-math3, particle_core, placeholder-api, reeses-sodium-options, satin, skyblocker, sneakfix, sodium, sodium-extra, titletweaks, tooltipscroll, tweakeroo, vmp, volumefix, wrench_wrapper, wurst, yet_another_config_lib_v3, file/yingcan, aaron-mod:chroma_text

-- System Details --
Details:
	Minecraft Version: 1.21
	Minecraft Version ID: 1.21
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 21.0.3, Eclipse Adoptium
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode, sharing), Eclipse Adoptium
	Memory: 297195640 bytes (283 MiB) / 1568669696 bytes (1496 MiB) up to 7730102272 bytes (7372 MiB)
	CPUs: 4
	Processor Vendor: GenuineIntel
	Processor Name: Intel(R) Core(TM) i5-4210U CPU @ 1.70GHz
	Identifier: Intel64 Family 6 Model 69 Stepping 1
	Microarchitecture: Haswell (Client)
	Frequency (GHz): 2.39
	Number of physical packages: 1
	Number of physical CPUs: 2
	Number of logical CPUs: 4
	Graphics card #0 name: Intel(R) HD Graphics Family
	Graphics card #0 vendor: Intel Corporation
	Graphics card #0 VRAM (MiB): 1024.00
	Graphics card #0 deviceId: VideoController1
	Graphics card #0 versionInfo: 20.19.15.5126
	Graphics card #1 name: AMD Radeon R7 M265 Series
	Graphics card #1 vendor: Advanced Micro Devices, Inc.
	Graphics card #1 VRAM (MiB): 2048.00
	Graphics card #1 deviceId: VideoController2
	Graphics card #1 versionInfo: 27.20.20913.2000
	Memory slot #0 capacity (MiB): 8192.00
	Memory slot #0 clockSpeed (GHz): 1.60
	Memory slot #0 type: DDR3
	Memory slot #1 capacity (MiB): 8192.00
	Memory slot #1 clockSpeed (GHz): 1.60
	Memory slot #1 type: DDR3
	Virtual memory max (MiB): 24316.23
	Virtual memory used (MiB): 9796.40
	Swap memory total (MiB): 8192.00
	Swap memory used (MiB): 423.61
	Space in storage for jna.tmpdir (MiB): available: 11910.82, total: 53030.00
	Space in storage for org.lwjgl.system.SharedLibraryExtractPath (MiB): available: 11910.82, total: 53030.00
	Space in storage for io.netty.native.workdir (MiB): available: 11910.82, total: 53030.00
	Space in storage for java.io.tmpdir (MiB): available: 5715.75, total: 61441.00
	Space in storage for workdir (MiB): available: 11910.82, total: 53030.00
	JVM Flags: 3 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmn1105m -Xmx7372m
	Fabric Mods: 
		aaron-mod: Aaron's Mod 1.3.1+1.21
		async-pack-scan: Async Pack Scan 0.2.5
		badoptimizations: BadOptimizations 2.1.4
		clientcommands: Client Commands 2.9.1
			betterconfig: BetterConfig 1.3
			clientarguments: clientarguments 1.9
			net_fabricmc_mapping-io: mapping-io 0.5.1
		cloth-config: Cloth Config v15 15.0.127
			cloth-basic-math: cloth-basic-math 0.6.1
		cursorcenteredfix: CursorCentered Fix 1.0
		dynamic_fps: Dynamic FPS 3.6.3
			net_lostluma_battery: battery 1.1.0
		fabric-api: Fabric API 0.102.0+1.21
			fabric-api-base: Fabric API Base 0.4.42+6573ed8cd1
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.68+b5597344d1
			fabric-biome-api-v1: Fabric Biome API (v1) 13.0.29+5bd9f1bcd1
			fabric-block-api-v1: Fabric Block API (v1) 1.0.22+0af3f5a7d1
			fabric-block-view-api-v2: Fabric BlockView API (v2) 1.0.10+6573ed8cd1
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.52+0af3f5a7d1
			fabric-client-tags-api-v1: Fabric Client Tags 1.1.15+6573ed8cd1
			fabric-command-api-v1: Fabric Command API (v1) 1.2.49+f71b366fd1
			fabric-command-api-v2: Fabric Command API (v2) 2.2.28+6ced4dd9d1
			fabric-commands-v0: Fabric Commands (v0) 0.2.66+df3654b3d1
			fabric-content-registries-v0: Fabric Content Registries (v0) 8.0.16+b5597344d1
			fabric-convention-tags-v1: Fabric Convention Tags 2.0.19+7f945d5bd1
			fabric-convention-tags-v2: Fabric Convention Tags (v2) 2.5.0+c5e2b5c6d1
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.29+0af3f5a7d1
			fabric-data-attachment-api-v1: Fabric Data Attachment API (v1) 1.1.27+6a6dfa19d1
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 20.2.16+16c4ae25d1
			fabric-dimensions-v1: Fabric Dimensions API (v1) 4.0.0+6fc22b99d1
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.6.12+6fc22b99d1
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.7.12+ba9dae06d1
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.53+6ced4dd9d1
			fabric-item-api-v1: Fabric Item API (v1) 11.0.0+afdfc921d1
			fabric-item-group-api-v1: Fabric Item Group API (v1) 4.1.4+78017270d1
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.47+0af3f5a7d1
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.45+df3654b3d1
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.3.12+6c1df360d1
			fabric-loot-api-v2: Fabric Loot API (v2) 3.0.14+3f89f5a5d1
			fabric-loot-api-v3: Fabric Loot API (v3) 1.0.2+3f89f5a5d1
			fabric-message-api-v1: Fabric Message API (v1) 6.0.13+6573ed8cd1
			fabric-model-loading-api-v1: Fabric Model Loading API (v1) 2.0.0+fe474d6bd1
			fabric-networking-api-v1: Fabric Networking API (v1) 4.2.2+60c3209bd1
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 15.2.0+a551f7a4d1
			fabric-particles-v1: Fabric Particles (v1) 4.0.2+6573ed8cd1
			fabric-recipe-api-v1: Fabric Recipe API (v1) 5.0.12+65089712d1
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 5.1.2+60c3209bd1
			fabric-renderer-api-v1: Fabric Renderer API (v1) 3.4.0+c705a49cd1
			fabric-renderer-indigo: Fabric Renderer - Indigo 1.7.0+c705a49cd1
			fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.68+df3654b3d1
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.48+73761d2ed1
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.1.6+b5597344d1
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.71+df3654b3d1
			fabric-rendering-v1: Fabric Rendering (v1) 5.0.5+df16efd0d1
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 4.3.0+8dc279b1d1
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 1.3.0+56599129d1
			fabric-screen-api-v1: Fabric Screen API (v1) 2.0.24+b5597344d1
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.86+b5597344d1
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.23+6573ed8cd1
			fabric-transfer-api-v1: Fabric Transfer API (v1) 5.1.16+3dccd343d1
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 6.1.0+176f9036d1
		fabric-language-kotlin: Fabric Language Kotlin 1.11.0+kotlin.2.0.0
			org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 2.0.0
			org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 2.0.0
			org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 2.0.0
			org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 2.0.0
			org_jetbrains_kotlinx_atomicfu-jvm: atomicfu-jvm 0.24.0
			org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.8.1
			org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.8.1
			org_jetbrains_kotlinx_kotlinx-datetime-jvm: kotlinx-datetime-jvm 0.6.0
			org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.6.3
			org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.6.3
			org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.6.3
		fabricloader: Fabric Loader 0.15.11
		faster-random: Faster Random 5.1.0
		fastitems: Fast Items 1.0.0-1.21
			midnightlib: MidnightLib 1.5.7
		figura: Figura 0.1.5-rc.1+1.21-8256be1
			com_github_figuramc_luaj_luaj-core: luaj-core 3.0.8-figura
			com_github_figuramc_luaj_luaj-jse: luaj-jse 3.0.8-figura
			com_neovisionaries_nv-websocket-client: nv-websocket-client 2.14
			mixinextras: MixinExtras 0.3.6
		freelook: Freelook 1.2.2
		fzzy_config: Fzzy Config 0.3.5+1.21
			blue_endless_jankson: jankson 1.2.3
			net_peanuuutz_tomlkt_tomlkt-jvm: tomlkt-jvm 0.3.7
		higher-chat: Higher Chat 1.5
		java: OpenJDK 64-Bit Server VM 21
		jjpizza: Jumper's Pizzeria 1.5.0-test
		libjf: LibJF 3.16.0
			libjf-base: LibJF Base 3.16.0
			libjf-config-commands: LibJF Config Commands 3.16.0
			libjf-config-core-v2: LibJF Config 3.16.0
			libjf-config-network-v0: LibJF Config: Network 3.16.0
			libjf-config-ui-tiny: LibJF Config UI: Tiny 3.16.0
			libjf-data-manipulation-v0: LibJF Data Manipulation 3.16.0
			libjf-data-v0: LibJF Data 3.16.0
			libjf-mainhttp-v0: LibJF MainHTTP 3.16.0
			libjf-translate-v1: LibJF Translate 3.16.0
			libjf-unsafe-v0: LibJF Unsafe 3.16.0
			libjf-web-v1: LibJF Web 3.16.0
		lithium: Lithium 0.12.7
		malilib: MaLiLib 0.19.999-sakura.4
		methane: Methane 3.8.2
			libmcdev-fabric: libmcdev-fabric 1.4.2
		minecraft: Minecraft 1.21
		modelfix: Model Gap Fix 1.21-1.6
		modernfix: ModernFix 5.19.0+mc1.21
		modmenu: Mod Menu 11.0.1
		moreculling: More Culling 0.26.0
			com_github_fallen-breath_conditional-mixin: conditional-mixin v0.6.2
			mixinsquared: MixinSquared 0.1.1
		motionblur: Motion Blur 1.0.5+mc1.21
		ok_zoomer: Ok Zoomer 9.2.0
			libzoomer: LibZoomer 0.10.0+1.21
			wrench_wrapper: Wrench Wrapper 0.4.0
		particle_core: Particle Core 0.2.4+1.21
			conditional-mixin: conditional mixin 0.5.1
		reeses-sodium-options: Reese's Sodium Options 1.7.3+mc1.21
		satin: Satin 2.0.0
		screenshotclipboard: Screenshot to Clipboard 1.0.9
		servercore: ServerCore 1.5.4+1.21
			fabric-permissions-api-v0: fabric-permissions-api 0.3.1
			org_yaml_snakeyaml: snakeyaml 2.2
			placeholder-api: Placeholder API 2.4.0+1.21
			space_arim_dazzleconf_dazzleconf-core: dazzleconf-core 1.3.0-M2
			space_arim_dazzleconf_dazzleconf-ext-snakeyaml: dazzleconf-ext-snakeyaml 1.3.0-M2
		skyblocker: Skyblocker 1.21.1-beta-ac08fb6+1.21.1
			com_logisticscraft_occlusionculling: occlusionculling 0.0.8-SNAPSHOT
			hm-api: HM Api 1.0.0+1.21
			meteordevelopment_discord-ipc: discord-ipc 1.1
			moe_nea_neurepoparser: neurepoparser 1.5.0
			net_azureaaron_networth-calculator: networth-calculator 1.0.0
			org_apache_commons_commons-math3: commons-math3 3.6.1
			org_apache_commons_commons-text: commons-text 1.12.0
			org_eclipse_jgit_org_eclipse_jgit: org.eclipse.jgit 6.10.0.202406032230-r
		sneakfix: SneakFix 1.0.0
		sodium: Sodium 0.5.11+mc1.21
		sodium-extra: Sodium Extra 0.5.7+mc1.21
			caffeineconfig: CaffeineConfig 1.3.0+1.17
		titletweaks: Title Tweaks 1.1.2
		tooltipscroll: Tooltip Scroll 1.3.0+1.21
		tweakeroo: Tweakeroo 0.20.999-sakura.8
		vmp: Very Many Players 0.2.0+beta.7.163+1.21
			com_ibm_async_asyncutil: asyncutil 0.1.0
		volumefix: Volume Fix 1.2.0
		wurst: Wurst Client v7.45-MC1.21.1
		yet_another_config_lib_v3: YetAnotherConfigLib 3.5.0+1.21-fabric
			com_twelvemonkeys_common_common-image: common-image 3.10.0
			com_twelvemonkeys_common_common-io: common-io 3.10.0
			com_twelvemonkeys_common_common-lang: common-lang 3.10.0
			com_twelvemonkeys_imageio_imageio-core: imageio-core 3.10.0
			com_twelvemonkeys_imageio_imageio-metadata: imageio-metadata 3.10.0
			com_twelvemonkeys_imageio_imageio-webp: imageio-webp 3.10.0
			org_quiltmc_parsers_gson: gson 0.2.1
			org_quiltmc_parsers_json: json 0.2.1
	Launched Version: 1.21-Fabric 0.15.11
	Launcher name: PCL
	Backend library: LWJGL version 3.3.3-snapshot
	Backend API: Intel(R) HD Graphics 4400 GL version 3.2.0 - Build 20.19.15.5126, Intel
	Window size: 1366x768
	GFLW Platform: win32
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Is Modded: Definitely; Client brand changed to 'fabric'
	Universe: 400921fb54442d18
	Type: Client (map_client.txt)
	Graphics mode: fast
	Render Distance: 2/2 chunks
	Resource Packs: vanilla, fabric, aaron-mod, async-pack-scan, betterconfig, caffeineconfig, clientcommands (incompatible), cloth-config, conditional-mixin, dynamic_fps, fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-blockrenderlayer-v1, fabric-client-tags-api-v1, fabric-command-api-v1, fabric-command-api-v2, fabric-commands-v0, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-convention-tags-v2, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-game-rule-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-keybindings-v0, fabric-language-kotlin, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-loot-api-v3, fabric-message-api-v1, fabric-model-loading-api-v1, fabric-networking-api-v1, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-renderer-registries-v1, fabric-rendering-data-attachment-v1, fabric-rendering-fluids-v1, fabric-rendering-v0, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-sound-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader, faster-random, fastitems, figura, freelook, fzzy_config, higher-chat, hm-api, jjpizza, libjf, libjf-config-core-v2, libjf-config-network-v0, libjf-config-ui-tiny, libjf-data-v0, libjf-translate-v1, libjf-web-v1, libmcdev-fabric, libzoomer, lithium, malilib, methane, midnightlib, modernfix (incompatible), modmenu, moreculling, motionblur, ok_zoomer, org_apache_commons_commons-math3, particle_core, placeholder-api, reeses-sodium-options, satin, skyblocker, sneakfix, sodium, sodium-extra, titletweaks, tooltipscroll, tweakeroo (incompatible), vmp, volumefix, wrench_wrapper, wurst, yet_another_config_lib_v3 (incompatible), file/yingcan (incompatible), aaron-mod:chroma_text
	Current Language: zh_cn
	Locale: zh_CN
	System encoding: GBK
	File encoding: UTF-8
	CPU: 4x Intel(R) Core(TM) i5-4210U CPU @ 1.70GHz
```

**Comment by kevinthegreat1** [comment]
Hypixel why did you put a pet there 😭

**Comment by x-yingcan-x** [comment]
btw, is it possible to launch with local repo and try update ingame, failling to download repo bothered me many times

**Comment by kevinthegreat1** [comment]
> btw, is it possible to launch with local repo and try update ingame, failling to download repo bothered me many times

I think if you download [the repo](https://github.com/NotEnoughUpdates/NotEnoughUpdates-REPO) and put it in `/config/skyblocker/item-repo` it should work.

**Comment by x-yingcan-x** [comment]
the question is, its always there but never work, i tried delete it and re-download, still not work at all

**Comment by kevinthegreat1** [comment]
Can you try #921 please.

**Comment by x-yingcan-x** [comment]
tried #920 , works well

**Comment by AzureAaron** [comment]
> goodness, tried pet fix, another crash, [crash-2024-08-13_23.03.11-client.txt](https://github.com/user-attachments/files/16601497/crash-2024-08-13_23.03.11-client.txt) [latest.log](https://github.com/user-attachments/files/16601498/latest.log) for short, NPE, caused by nw calc

Thanks for reporting :) Fixed in #928
